integration/integration_test.go

```diff
@@ -53,14 +55,15 @@ func Test(t *testing.T) {
 		return
 	}
 
+	check.Suite(&ACLDisabledSuite{})
```

Comment:
Since you modified the suites, you should probably also update `.semaphore/semaphore.yaml` so that they run in the CI 😝